src/service/packet_router.proto

```diff
@@ -52,10 +52,38 @@ message packet_router_register_v1 {
   bytes signature = 3;
 }
 
+// Session offer from the packet router to the gateway. If the gateway wants to
```

Comment:
I think `bytes gateway = 9;` in `packet_router_packet_up_v1` above should signal that it is not always the gateway but the public session key, no?